modules/database/funcs.go

```diff
@@ -229,3 +232,36 @@ func (d *Database) getDBWithName(
 
 	return nil
 }
+
+//
+// Returns a *Database object with specified name and namespace
+//
+func DatabaseName(
```

Comment:
I liked the previous `GetDatabaseByName` name for this function.

Reply:
ok I'll change it to GetDatabaseByName

Reply:
also thanks for the git explanation , some of it I was less sure of